src/scripts/api.ts

```diff
@@ -246,4 +246,23 @@ export const fetchRanks = async (id: string, auth: Credentials) => {
   } catch (error) {
     return handleAxiosError(AxiosError.from(error));
   }
-}
\ No newline at end of file
+}
+
+interface OrganizationData {
+  name: string;
+  display_name: string;
+  description: string;
+}
+
+export const CreateOrganization = async (auth: Credentials, form: OrganizationData) => {
```

Comment:
```suggestion
export const createOrganization = async (auth: Credentials, form: OrganizationData) => {
```

Consider use camel case naming convention.